client/src/templates/Introduction/components/block.tsx

```diff
@@ -82,16 +96,6 @@ class Block extends Component<BlockProps> {
     );
   }
 
-  renderBlockIntros(arr: string[]): JSX.Element {
-    return (
-      <div className='block-description'>
-        {arr.map((str, i) => (
-          <p dangerouslySetInnerHTML={{ __html: str }} key={i} />
-        ))}
-      </div>
-    );
-  }
-
```

Comment:
We need to parse the Markup:

<img width="521" alt="image" src="https://user-images.githubusercontent.com/51722130/229491300-8105fe19-3616-4450-9c55-0237d5c75e7b.png">